src/FileExplorer.js

```diff
@@ -104,11 +106,15 @@ function FileExplorer(props) {
 
         function open()
         {
-            // its important to set the current path first !!
-            getFilesFromResourceURL(url).then((fileArray) => {
-                setCurrentPath(url);
-                setFiles(fileArray);
-            });
+            if (url[url.length - 1] === "/") {
```

Comment:
.endsWith("/") exist in JS - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith - does the same but generally more readable

Reply:
very good idea !